fdbserver/TLogGroup.actor.cpp

```diff
@@ -127,6 +211,54 @@ void TLogGroupCollection::loadState(const Standalone<RangeResultRef>& store) {
 	}
 }
 
+void TLogGroupCollection::seedTLogGroupAssignment(Arena& arena,
+                                                  CommitTransactionRef& tr,
+                                                  vector<StorageServerInterface> servers) {
+
+	// Collect UID of SS
+	std::vector<UID> serverSrcUID;
+	serverSrcUID.reserve(servers.size());
+	for (auto& s : servers) {
+		serverSrcUID.push_back(s.id());
+	}
+
+	// Step 1: Create the first storage server team.
+	auto teamId = deterministicRandom()->randomUniqueID();
+	tr.set(arena, storageTeamIdKey(teamId), encodeStorageTeams(serverSrcUID));
+
+	// Step 2: Map from SS to teamID.
+	for (const auto& ss : serverSrcUID) {
+		Key teamIdKey = storageServerToTeamIdKey(ss);
+		BinaryWriter wr(Unversioned());
+		wr << 1;
+		wr << teamId;
+		tr.set(arena, teamIdKey, wr.toValue());
+	}
+
+	// Step 3: Assign the storage team to a TLogGroup (Storage Team -> TLogGroup).
+	TLogGroupRef group = selectFreeGroup();
+	TraceEvent("TLogGroupSeedTeam").detail("StorageTeamId", teamId);
+	tr.set(arena, storageTeamIdToTLogGroupKey(teamId), BinaryWriter::toValue(group->id(), Unversioned()));
+	assignStorageTeam(teamId, group);
+	storageTeams[teamId] = serverSrcUID;
+}
+
+// Returns a map from StorageTeamID that uniquely identifies a team, to the
+// list of storage servers in that storage team.
+ACTOR Future<std::map<ptxn::StorageTeamID, std::vector<UID>>> fetchStorageTeams(IKeyValueStore* store) {
```

Comment:
Will this function be used anywhere? If `master` calls this, I think this can be a non-actor function, because `store->readRange` is not blocking.